libs/sdk-backend-tiger/src/backend/workspaces/index.ts

```diff
@@ -52,8 +53,9 @@ class TigerWorkspaceQuery implements IWorkspacesQuery {
         return this;
     }
 
-    public withParent(workspaceId: string): IWorkspacesQuery {
-        this.parentWorkspaceId = workspaceId;
+    public withParent(workspaceId: string | undefined): IWorkspacesQuery {
+        // isnull=true - get only root workspaces
```

Comment:
Please add the following text as a comment so that people know when the function should not be called.

```
This function should not be called when we want to fetch all workspaces (root and non-root ones)
```

Reply:
fixed